packages/rocketchat-ui-sidenav/client/roomList.js

```diff
@@ -13,7 +13,17 @@ Template.roomList.helpers({
 		if (this.anonymous) {
 			return RocketChat.models.Rooms.find({t: 'c'}, {sort: {name: 1}});
 		}
-		const user = Meteor.userId();
+
+		const user = RocketChat.models.Users.findOne(Meteor.userId(), {
```

Comment:
`user` was a string, the result of `Meteor.userId()`. Can you confirm how `user` is used in this scope?